### PR TITLE
#20903 Created paddle.tensor.math.stanh function and test

### PR DESCRIPTION
**Pull request by davidMadueke**
Created the paddle.tensor.math.stanh function out of composite Ivy functions. According to the native framework documentation the function takes three arguments X (the native or ivy array) and two floats (scale_a and scale_b) which are nominally 0.67 and 1.7159 according to the stanh documentation (see: https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/stanh_en.html )

-Note- as of this commit, I was struggling to get the implemented functions to pass all of the tests. Whenever the test will run, the function will return an array with value NAN for test cases which will make all of the tests fail. But as of today (05 AUG 2023) I came across a TODO comment made by Haris Mahmood on the paddle.tensor.linalg.pinv frontend test where they detailed how they were experiencing similar bugs and when I ran the test for that function, it seemed to fail in the same manner as my implementation.

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ZoeCD** [comment]
Thank you, David!
In that case, we can merge your PR, since it is not a problem with your implementation. But, I need to ask you two things.
1) Please create an issue for this PR as instructed [here](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues). It helps us keep track of what's done
2) Solve the merge conflicts in your branch and push the changes
Thanks!

**Comment by davidMadueke** [comment]
Ok thanks for the response

Close #20903

**Comment by davidMadueke** [comment]
Hi, I was running the automatic tests and found that the Lint / Check formatting / check-formatting (pull_request) was failing. I proceeded to check the extra details on the problem and the failing test mentioned the label "black". I am not too sure what this means, is there something specific related to my commit that I need to change. Or is it somthing not necessarily related to my implementation
image

Also, once I get this problem resolved, I am not too sure how to close this pull request. Do I press the Close with Comment button at the bottom of this PR. Sorry if this is a basic thing to do. I am fairly new to contributing to open source projects :) .

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py [hunk 1]
```diff
@@ -2037,14 +2038,34 @@ def test_paddle_amax(
     frontend,
     test_flags,
 ):
+  
+
+=======
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py:2043]: Hello again! You will not be able to merge the PR yourself. Only reviewers can do that. Feel free to ask whatever you need, we are here to help!

Also, it seems like there are some residual lines from when you merged the new changes. Maybe this is what the lint test was flagging. I recommend that you copy and save your changes in a text editor so you don't lose them, and update your branch to restore the file. Then, you can add your changes again if you need to, or simply push everything again

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py (hunk 1 above)
Comment (davidMadueke) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py]: I think I resolved the changes by making a new set of commits

**Comment by davidMadueke** [comment]
Hey Zoe, I just made the changes you requested and cleaned up my branch so that it no longer caused any merge conflicts. However, Github is still saying 1 change requested again. I am using Pycharm with Fork to make and commit all of my changes. Is it something related on my end to change.

image

**Comment by davidMadueke** [comment]
> Hey Zoe, I just made the changes you requested and cleaned up my branch so that it no longer caused any merge conflicts. However, Github is still saying 1 change requested again. I am using Pycharm with Fork to make and commit all of my changes. Is it something related on my end to change.
> 
> image

Sorry, I just figured out what I did wrong. Turns out in the FORK Gui it allows for one to skip the pre-commit hooks as they create their commit which I was doing. Once I cancelled the request to skip it, the BLACK formatter would automatically change the code giving me the option to stage that change and commit that making everything work well again

image

**Comment by ZoeCD** [comment]
Lgtm now! Thank you!

**Pull request merged**